src/libs/ReceiptUtils.ts

```diff
@@ -63,6 +64,10 @@ function getThumbnailAndImageURIs(transaction: Transaction, receiptPath: string
         image = ReceiptSVG;
     }
 
+    if (fileExtension === CONST.IOU.FILE_TYPES.PDF) {
+        return {thumbnail: image, image: path, isLocalFile: true};
+    }
+
```

Comment:
@pradeepmdk Do we support previewing SVG and HTML files? What is the result when we upload SVG, HTML and DOC file and try to preview them?

Reply:
html and doc will converted as a pdf but when we upload svg it will throw a error 
<img width="1184" alt="Screenshot 2023-11-07 at 1 08 07 PM" src="https://github.com/Expensify/App/assets/16595846/7e2295df-88b5-4439-a07a-ee8001f38eba">

Reply:
@pradeepmdk In that case, if I am not wrong we will still see "Unable to load pdf file" for doc and html file. I prefer showing pdf for those file types as well. If we won't have any issue with other file extensions, can we make it the final return of the fuction.

Reply:
@sobitneupane Yes we can see "Unable to load pdf file" until the pusher sends the updated pdf path. and yes we can make as a final return. until the pusher sends the data it will show the local image. once we receive it will automatically load the pdf.

Reply:
let me make as a final return.